connectivity/netsocket/tests/TESTS/network/wifi/wifi_connect_disconnect_repeat.cpp

```diff
@@ -34,8 +34,10 @@ void wifi_connect_disconnect_repeat(void)
     TEST_ASSERT_EQUAL(NSAPI_ERROR_OK, error);
 
     for (int i = 0; i < 10; i++) {
+        printf("#%u connecting...\n", i);
```

Comment:
Can you add separate commit for adding these debug messages (I would like to understand why are they being added)